### ISSUE TITLE
Links for "AM implementation and pretrained model" aren't accessible

### ISSUE THREAD

**Issue by mountains-high** [currybur/CS222-Class-Adaptive-Pruning#1]
Hi~
Thank you for providing great work.
I wanted to use the pretrained model, but links seem not accessible now. Could you provide again, please?
I tried to train from the scratch but test accuracy was low.

Thank you

**Comment by currybur** [comment]
Hi, thanks for your interest.

The pre-trained model is the official VGG-19 network pre-trained on ImageNet and is available [here](https://download.pytorch.org/models/vgg19-dcbb9e9d.pth). Since this repo uses cifar-10 to generate activation features, we train a network for a few epochs on cifar-10 based on the pre-trained weights, and this may not take long. 

Besides, if you would like to delve deeper, I strongly recommend this [repo](https://github.com/david990917/Model-Pruning-in-Deep-Learning), which has successfully pruned some weights.

Regards

**Comment by mountains-high** [comment]
Hi ~
Thank you very much for your response.
I will also study recommended repository. 

Have a nice day!
Thank you
